src/foam/nanos/auth/LookAndFeel.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'LookAndFeel',
```

Comment:
Personally this feels more like a Theme than LookAndFeel.  I also don't think it should live the auth package, its only loosely auth related.

Reply:
I agree with both points.

Reply:
Resolved by 4fdb80c.